lib/cf/CloudControllerClient.js

```diff
@@ -70,6 +70,10 @@ class CloudControllerClient extends HttpClient {
     return this.getResource(`/v2/service_instances/${guid}`, options);
   }
 
+  getService(guid, options) {
```

Comment:
Method not required. All the service & plan level information for a guid are within service fabrik itself. Refer methods in `catalog.js`